numpy/core/memmap.py

```diff
@@ -3,7 +3,7 @@
 import numpy as np
```

Comment:
Can you backout the changes in this file and save them for a separate, ENH, PR? This is going beyond internal cleanup to (good!) behavior changes.